src/SingleBookingPage/bookingItem/AccordionLegCities.js

```diff
@@ -137,11 +138,6 @@ class LegCities extends React.Component<Props, State> {
 
     const { isExpanded } = this.state;
 
-    const legTitle =
```

Comment:
in my opinion, it would be cleaner and more readable, if the logic behind the leg title would remain here, and in the return we would just use this computed value